src/app/screens/Home/index.tsx

```diff
@@ -39,17 +42,34 @@ function Home() {
       });
       const [currentTab] = tabs;
       const url = new URL(currentTab.url as string);
+      setCurrentUrl(url);
       const result = await api.getAllowance(url.host);
       if (result.enabled) {
         setAllowance(result);
       }
+      const blocklistResult = await api.getBlocklist(url.host);
+      if (blocklistResult.blocked) {
+        setBlocklist(blocklistResult.blocked);
+      }
     } catch (e) {
       console.error(e);
     } finally {
       setLoadingAllowance(false);
     }
   }
 
+  async function unblock() {
+    try {
+      if (currentUrl?.host)
+        await utils.call("deleteBlocklist", {
+          host: currentUrl.host,
+        });
+      window.location.reload(); // refresh home page
```

Comment:
why do we need this actually? isn't it enough to set `blocklist` to null?

Reply:
this is needed so the extension UI will reset and `unblock` button will disappear.

Reply:
but we have this check here: https://github.com/getAlby/lightning-browser-extension/pull/825/files#diff-041371f0bc0d8e5acd8d60f55e0b217ee4320404c44f5ebe9a3d61db95753a45R254

isn't then `setBlocklist(null)` enough? the `blocklist` is falsy and that button should disappear.

Reply:
I've tested again and indeed it doesn't automatically disappear. reload is needed, so I'm leaving this in.

Reply:
`setBlocklist(false);` seems to work fine or am I missing something?

Reply:
works fine for me, too. I did `setBlocklist(null)` - as `null` is the default value and also specified in the type.